tests/enforcer/gce_firewall_enforcer_test.py

```diff
@@ -865,7 +1038,7 @@ def test_build_change_set(self):
                 u'Allow communication between instances.',
             'name':
                 u'unknown-rule-doesnt-match',
-            'network': (u'https://www.googleapis.com/compute/v1/projects/'
+            'network': (u'https://www.googleapis.com/compute/beta/projects/'
                         'google.com:secops-testing/global/networks/test-net'),
```

Comment:
Please ensure all references to google.com:secops-testing or other test projects are replaced with forseti-system-test.

Reply:
Done